src/js/facility-locator/utils/helpers.js

```diff
@@ -0,0 +1,76 @@
+export const keyMap = {
+  TAB: 9,
+  ENTER: 13,
+  ESCAPE: 27,
+  SPACE: 32,
+  UP: 38,
+  DOWN: 40
+};
+
+export function getDirection(code) {
+  if (code === keyMap.UP) return -1;
+  if (code === keyMap.DOWN) return 1;
+  return false;
+}
+
+export function getSelection(options, queryType) {
+  let selection = options[0];
+  if (queryType) {
+    const selectionType = queryType;
+    selection = options.reduce((acc, elem) => {
```

Comment:
This seems more like `find` than reduce:

```
selection = options.find(elem => elem.id === selectionType) || selection;
```

Reply:
Good point.